### ISSUE TITLE
HTML email content not escaped

### ISSUE THREAD

**Issue by nriley** [jeffaco/duplicacy-util#3]
There are several places where text is included in the HTML content without escaping it, which can cause text to seemingly disappear.  For example:

`21:05:22 duplicacy-util starting, version: <dev>, Git Hash: <unknown><br>
`

I have next to zero Go experience but I do see the standard library includes a HTML templating package; is there a reason why you don't use it?

Thanks for duplicacy-util and the inspiration to use Gmail for backup log aggregation!

**Comment by jeffaco** [comment]
Yup, this is indeed on my list of things to do, and indeed the standard Go library supports this. I didn't include it out of the gate since I was barely familiar with HTML before adding HTML support to `duplicacy-util`, so I wanted to take baby steps and get HTML working.

While I absolutely plan on doing HTML escaping, part of why I don't have it yet is because I didn't see any issues without it. That is, until @nriley pointed one out! 😞 I'll look at this soon, and will implement it before doing checkpointing (which I really want to do since I recently had a `duplicacy` problem where that would have been nice).

FWIW, the bug that you saw above was due to an unrelated build bug, which is already fixed.. The `<dev>` and `<unknown>` fields should have been filled in, which would have hidden this bug. That problem is already fixed in master, and looks like it was tagged with `v1.1`. Does `v1.1` show that problem, or were you running a prior version or build from source or something?

I'll resolve this issue once HTML escaping is incorporated. I'll work on this soon.

**Comment by nriley** [comment]
Thanks! I was building from source (due to my inability to find binaries for 1.1, i.e. #1 :-)

**Comment by jeffaco** [comment]
Fixed in commit 44a15e4. This will be in the next released version, or you can build from source to get the fix sooner. I don't think this is pressing, so I won't re-release just for that.

A good way to test this is to just build from source and send E-Mail. The versioning will get messed up in the E-Mail log output, and will appear as:

```
18:05:10 duplicacy-util starting, version: , Git Hash: 
```

This should appear as:

```
18:05:10 duplicacy-util starting, version: <dev>, Git Hash: <unknown>
```

Additionally, logging was enhanced to show how much data was backed up in the log a while back, like this:

```
18:55:28 Backing up to storage b2 with 10 threads
18:55:31   Files: 345 total, 823,165K bytes; 0 new, 0 bytes
18:55:31   All chunks: 150 total, 890,186K bytes; 0 new, 0 bytes, 0 bytes uploaded
18:55:31   Duration: 3 seconds
18:55:31 Backing up to storage azure-direct with 5 threads
18:55:32   Files: 345 total, 823,165K bytes; 0 new, 0 bytes
18:55:32   All chunks: 150 total, 889,922K bytes; 0 new, 0 bytes, 0 bytes uploaded
18:55:32   Duration: 1 second
18:55:32 Copying from storage b2 to storage azure with 10 threads
18:55:35   Copy complete, 110 total chunks, 0 chunks copied, 110 skipped
18:55:35   Duration: 3 seconds
```

I used non-breaking spaces to preserve multiple spaces in the HTML output (output from `backup` or `copy` commands).

**Comment by nriley** [comment]
Works for me now.  Thanks.
